docs/current_docs/features/visualization.mdx

```diff
@@ -0,0 +1,75 @@
+- `-vv` reveals internal and encapsulated spans;
+- `-vvv` reveals sub-100ms spans.
+
+For additional debugging information, add the `--debug` flag to the `dagger call` command.
+
+## Traces
+
+Dagger Cloud provides Traces, a browser-based interface focused on tracing and debugging Dagger pipeline runs. A Trace represents one invocation of a Dagger Function, run locally or in CI. It contains detailed information about the operations performed by the Dagger Function.
+
+Traces provides a UI to debug what went wrong when something fails, focusing on tracing what happened and when it happened. Traces also helps you optimize your Dagger Functions as you author them, by giving you the detailed step-by-step execution of your pipeline and the associated telemetry. This is equally useful for debugging pipelines running on a developer's local environment and for pipelines in CI environments.
+
+Here's an example of it in action:
+
+![Dagger Trace](/img/current_docs/quickstart/trace.gif)
+
+Dagger is aware of the context it runs on. The Dagger Cloud dashboard classifies Traces depending on whether you triggered a run from your local development host or from your CI environment. You can select the appropriate view from the dashboard sidebar. Here's an example showing Traces generated in a CI environment:
+
+![Traces list](/img/current_docs/features/trace-list.png)
+
+This initial view displays groups of Traces that belong to the same point in Git history. The grouping is determined by the branch/reference combination where Dagger was triggered. This is useful when you need to inspect all the Traces emitted at a specific, pinned point in the repository's version history
+
+Here's a quick summary of what you'll see for each Trace group:
+
+- Repository name
+- Commit metadata, including comment and commit author
+- Indication of the run status: in progress, succeeded, or failed
+- The command used
+- The run start time
+- The run duration
+
+:::important
+All of the information listed above may not be available for a specific group; it differs depending on whether the Trace is generated by a local run or a CI run. When Dagger runs in a CI context like GitHub, GitLab, CircleCI, or Jenkins, additional Trace metadata is displayed based on the Git repository information available.
+:::
+
+You can drill down into the details of a specific group by clicking it. This directs you to a list of Traces, together with detailed status and duration metadata for each. Each Dagger Function call is represented as a separate Trace, as shown below:
+
+![Trace details](/img/current_docs/features/trace-details.png)
+
+You can click any of the Traces in the list to see detailed logs for each Trace. The flame graph at the top of the Trace shows the operations performed by the Dagger Function, with a visual cue of how much time each operation represents of the total duration, as shown below:
+
+![Trace log](/img/current_docs/features/trace-log.png)
+
+:::info
```

Comment:
IMHO this could be sent to the top of the Traces section, as it gives a bit of context of why we call them Traces. OTEL is a kinda well- know protocol, so it will likely resonate among DevOps oriented users.

Reply:
Done